Sources/Foundation/NSStringAPI.swift

```diff
@@ -14,58 +14,12 @@
 //
 //===----------------------------------------------------------------------===//
 
-// Important Note
-// ==============
-//
-// This file is shared between two projects:
-//
-// 1. https://github.com/apple/swift/tree/master/stdlib/public/Darwin/Foundation
```

Comment:
This file no longer exists / this is no longer the case, so we can remove this comment altogether